sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_custom_documents.py

```diff
@@ -124,6 +127,40 @@ def analyze_custom_documents(custom_model_id):
                                 row_data = list(map(extract_value_by_col_name, col_names))
                                 table_rows.append(row_data)
                         print_table(col_names, table_rows)
+                    
+                    elif (
+                        field_value.type == SYMBOL_OF_OBJECT_TYPE
+                        and field_value[KEY_OF_VALUE_OBJECT] is not None
+                    ):
+                        rows_by_columns = list(field_value[KEY_OF_VALUE_OBJECT].values())
+                        is_fixed_table = all(
+                            (
+                                rows_of_column["type"] == SYMBOL_OF_OBJECT_TYPE
+                                and Counter(
```

Comment:
curious, why don't we use `len`?

Reply:
Due to the data about "Row" storage as  object.  Use Counter() to ensure key's length and string are equal more precisely.
![RowData](https://github.com/TakaValley/azure-sdk-for-python/assets/26052497/63839c93-0711-4727-a304-907280693c3e)